src/jdk.compiler/share/classes/com/sun/tools/javac/resources/javac.properties

```diff
@@ -383,6 +383,8 @@ javac.msg.resource=\
 \n\nThe system is out of resources.\n\
 Consult the following stack trace for details.\n
 
+javac.description=Read Java declarations and compile them into class files
+
```

Comment:
for your general consideration, what grammatical style do you recommend here? Should it be 3rd person (Reads) instead of 2nd person (Read)

Reply:
I took the phrases in 2nd person style from https://docs.oracle.com/en/java/javase/18/docs/specs/man/ as templates, shortening some of them to around 80 characters for better fitting in standard terminal widths.

Perhaps, those texts should be used "as-is" as descriptions?
- 2nd person phrase-style
- lowercase start
- no terminating period
- no shortening

Reply:
Took the phrases "as-is" from the overview site for the man pages: [Java® Development Kit Version 18 Tool Specifications](https://docs.oracle.com/en/java/javase/18/docs/specs/man/)